### ISSUE TITLE
about augmenation in maskrcnn_input.py

### ISSUE THREAD

**Issue by spydr1** [tensorflow/models#10975]
https://github.com/tensorflow/models/blob/00b830c7ca641413f37e5147cf059c89f79b0e8e/official/vision/dataloaders/maskrcnn_input.py#L206

I'm not sure if you don't use augmentation on the mask r-cnn.
But if user tries to use augmentation, it won't apply to the bbox, so I think it'll work unintentionally.

https://github.com/tensorflow/models/blob/00b830c7ca641413f37e5147cf059c89f79b0e8e/official/vision/dataloaders/retinanet_input.py#L171
